README.md

```diff
@@ -107,19 +107,19 @@ This is the list of all currently supported blockchains (several coming soon and
 ## Configuration
 The agent loads its configuration by looking at the following sources in order:
 1. Load configuration `agent.yml` from `./configs` or current directory. The first to be found takes effect. If no files exist, proceed to step 2.
-2. Load configuration from environment variables and overload any parameter set.
+2. Load configuration from environment variables and overload any parameter set (except exporters).
```

Comment:
Was wondering why did you make this shortcut. This introduces an incosistency with regards to how the configuration file is treated by the agent and ideally we shouldn't have these cornercases.

Reply:
My reasoning behind this was such:

All other config parameters are static - we know them and their default values, each of them is a field on a struct. Exporter configuration, however, is implemented by the anyone who writes the exporters, we could arguably call it dynamic. I had 2 ideas on how to approach exporter configurations to begin with:

1. All configs reside in `contrib` package, we pass `any` type that contains all the configs, exporters extract that data as they see fit (e.g. `mapstructure.Decode`)  (my current implementation)
2. All configs are static - every new exporter needs to add a static configuration struct to `global` package.

In option 1 we cannot implement the environment variable overriding reliably - if there's no underlying or default value for the field we cannot correctly choose the type, not to mention a best effort implementation would be quite ugly, with the use of reflection and all.

If we went with option 2 however, every implementation of an exporter would need to add a struct type to `global` package and attach it under `Exporters` and it'd also be a requirement for exporter writers to handle the logic of default values and envvar overriding, which I believe is too big of an ask, not to mention it would really expand the size of `global` package.

Do you have any ideas on how to implement this or how to better approach it altogether?

Reply:
>Do you have any ideas on how to implement this or how to better approach it altogether?
Not really, I understand your rationale let's keep it as is.